client/src/components/graph/setupLasso.js

```diff
@@ -10,6 +10,7 @@ const Lasso = () => {
     let lassoPolygon;
     let lassoPath;
     let closePath;
+    let lassoCompleted;
```

Comment:
It's more of a nit, but would it make more sense to name this something like `lassoInProgress` which gets initialized to `false` and then gets set to True in handleDragStart and False in handleDragEnd? Then the check you do in handleDragStart can remain the same (i.e. if lassoInProgress, cancel the existing one).

I just got a little confused with the naming around what having a lasso _complete_ meant in relation to prevent parallel lassos.